dev/app/builtin/stories/powered-by.stories.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/default */
+
+import { storiesOf } from 'dev-novel';
+import instantsearch from '../../../../index';
+import { wrapWithHits } from '../../utils/wrap-with-hits.js';
+
+const stories = storiesOf('PoweredBy');
+
+window.connectPoweredBy = instantsearch.connectors.connectPoweredBy;
+window.poweredBy = instantsearch.widgets.poweredBy;
```

Comment:
Why do we copy the reference on the global object?

Reply:
Oops! Forgot to remove those.